feature-group-definitions/page-breaks.dist.yml

```diff
@@ -0,0 +1,33 @@
+# Generated from: page-breaks.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: Page breaks
+description: The `break-after`, `break-before`, `break-inside` CSS properties (along with `page-break-` aliases) control where printed pages start and end. Also known as pagination or page breaking.
+spec: https://drafts.csswg.org/css-break-3/#breaking-controls
+caniuse: css-page-break
```

Comment:
https://caniuse.com/css-page-break says partial support across the board, except good old Presto of course. Can you add a comment about the difference and if there's anything we need TODO?

Reply:
Ah, this. Caniuse says everything is partially implemented. Some of this is just out-of-date (e.g., lack of support for `break-` properties)—I can open a PR against caniuse for that.

But because nothing is shown as being fully supported on caniuse except Presto, we can't be sure what Alexis meant by fully-supported. I've filed https://github.com/Fyrd/caniuse/issues/7045.

The good news here is that I believe we still ought to show a positive status. If caniuse says partially supported, it's still _equally_ partially implemented across all browsers. They seem to have the same set of limitations.

Reply:
> The good news here is that I believe we still ought to show a positive status. If caniuse says partially supported, it's still _equally_ partially implemented across all browsers. They seem to have the same set of limitations.

I spoke too soon. 😞 

Alexis tested this again and showed that the avoid value is not consistently supported by browsers. I submitted https://github.com/mdn/browser-compat-data/pull/22987 to update the BCD and changed the list of compat keys here to match caniuse.

This makes me think that the multicolumn breaks might also be broken, but I couldn't actually find bugs to that effect, so I'm letting that stuff stand.

Reply:
Oh my, things are rarely simple. I've reviewed and merged that PR.